.buildkite/scripts/steps/functional/defend_workflows.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+.buildkite/scripts/bootstrap.sh
+node scripts/build_kibana_platform_plugins.js
```

Comment:
we should be able to use `.buildkite/scripts/download_build_artifacts.sh` instead here - they're shared from the Build distribution step

Reply:
Thank you @jbudz, but we are looking for using https://github.com/abhinaba-ghosh/cypress-react-selector in our tests, which requires unbundled Kibana code, so we can use React components as the targets (the same way we're using it currently in [Osquery Cypress tests ](https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/functional/osquery_cypress.sh)